sdk/tables/azure-data-tables/pom.xml

```diff
@@ -45,7 +45,7 @@ Licensed under the MIT License.
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-common</artifactId>
-      <version>12.9.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-common;current} -->
+      <version>12.9.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-common;current} -->
```

Comment:
@bsiegel you'll want to add a custom `beta_` tag for your dependency once this merges so Tables can depend on a released version.